modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/serviette.dm

```diff
@@ -5,6 +5,8 @@
 	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_items/lewd_items.dmi'
 	/// How much time it takes to clean something using it
 	var/cleanspeed = 5 SECONDS
+	/// Which item spawns after it's used
+	var/used_serviette = /obj/item/serviette_used
 	w_class = WEIGHT_CLASS_TINY
 	item_flags = NOBLUDGEON
 	gender = PLURAL
```

Comment:
One singular serviette isn't plural.
```suggestion
```

Reply:
Didn't intend to touch sex2 code that much but, fair enough.